### PR TITLE
Fast multi drag

### PR DESCRIPTION
**Pull request by pvh**
Bypass React entirely for updating cards during drag by using CSS variables. Ultimately, I think there's a better way involving a new Selection class but this gives us a much faster experience for now.

### CODE REVIEW

File: src/renderer/components/content-types/board/BoardCard.css [hunk 1]
```diff
@@ -23,6 +23,10 @@
   z-index: 2;
 }
 
+.BoardCardDragWrapper--selected {
+  translate: transform3d(var(--drag-x, 0), var(--drag-y, 0), 0)
```
Comment (pvh) [inline src/renderer/components/content-types/board/BoardCard.css]: this line should read `transform: translate3d`

**Comment by mjtognetti** [comment]
Seeing some interesting selection state drag behavior.

First, if a card is not already selected before it is dragged, the card does not appear to move with the drag, then snaps into position:
drag_behavior

Second, if a card is selected and another (not selected) card is dragged, the drag is applied to the selected card, not the card which is dragged:
drag_behavior_with_initial

**Comment by pvh** [comment]
Thanks! It's amazing what you can miss when you always test things the same way. This was happening because the add-to-selection dispatch was finishing after the callback in question. I've fixed it. Are you good to merge with the bug fixed?

**Comment by mjtognetti** [comment]
There's a few-pixel delay on starting a drag. Not sure if this is introduced here, but is notable nonetheless.

drag_delay

**Pull request merged**